src/app/CommandSender.h

```diff
@@ -114,6 +114,33 @@ class CommandSender final : public Messaging::ExchangeDelegate
         virtual void OnDone(CommandSender * apCommandSender) = 0;
     };
 
+    // While today there is only one item in this struct, this was added to make providing future elements of
```

Comment:
Does the single element imply that only one invoke request can be in a command? Or two? (original + additional)

Is the future plan to make this an array? If so, does having that in a struct actually help?

Reply:
I think this is better named as `AdditionalCommandDataElements`. There is expected to be one per command.

The problem I am looking to solve talked about in the above comment https://github.com/project-chip/connectedhomeip/pull/30675/files#r1407831738

Maybe I am overthinking it and should remove it all together and just pass what is needed when needed and let the parameter list grow if anything else shows up